cves/CVE-2010-1157.yml

```diff
@@ -202,4 +216,8 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    It seems that this vulnerability was an oversight to how the configuration and authentication code
```

Comment:
Grammar: Maybe change "an oversight to how" to "an oversight with regards to how"

Reply:
Yeah that makes more sense, thanks.